src/main/java/com/rakovets/course/java/core/practice/oop_principles/battle_ground/Zombie.java

```diff
@@ -0,0 +1,21 @@
+package com.rakovets.course.java.core.practice.oop_principles.battle_ground;
+
+public class Zombie extends Enemy {
+
+    public Zombie(int health) {
+        super(health);
+    }
+
+    @Override
+    public void attackHero(Hero hero) {
+        hero.takeDamageHero(20);
+    }
+
+    public int attackMag() {
```

Comment:
What is it?

Reply:
this is the method describing the attack by a zombie mage